util/measures/2019/ia-measures.csv

```diff
@@ -101,8 +101,8 @@ PSH Care Coordination,"Participation in a Perioperative Surgical Home (PSH) that
 Primary Care Physician and Behavioral Health Bilateral Electronic Exchange of Information for Shared Patients,The primary care and behavioral health practices use the same electronic health record system for shared patients or have an established bidirectional flow of primary care and behavioral health records.,IA_CC_16,Care Coordination,medium,,,,boolean,2018,,medium,careCoordination,
 Patient Navigator Program,"Implement a Patient Navigator Program that offers evidence-based resources and tools to reduce avoidable hospital readmissions, utilizing a patient-centered and team-based approach, leveraging evidence-based best practices to improve care for patients by making hospitalizations less stressful, and the recovery period more supportive by implementing quality improvement strategies.",IA_CC_17,Care Coordination,high,,,,boolean,2018,,high,careCoordination,
-Relationship-Centered Communication,"In order to receive credit for this activity, MIPS eligible clinicians must participate in a minimum of eight hours of training on relationship-centered care80 tenets such as making effective open-ended inquiries; eliciting patient stories and perspectives; listening and responding with empathy; using the ART (ask, respond, tell) communication technique to engage patients, and developing a shared care plan.",IA_CC_18,Care Coordination,medium,,,,boolean,2019,,medium,careCoordination,
-Use of certified EHR to capture patient reported outcomes,"The training may be conducted in formats such as, but not limited to: interactive simulations practicing the skills above, or didactic instructions on how to implement improvement action plans, monitor progress, and promote stability around improved clinician communication.",IA_BE_1,Beneficiary Engagement,medium,,,,boolean,2017,,medium,beneficiaryEngagement,
+Relationship-Centered Communication,"In order to receive credit for this activity, MIPS eligible clinicians must participate in a minimum of eight hours of training on relationship-centered care80 tenets such as making effective open-ended inquiries; eliciting patient stories and perspectives; listening and responding with empathy; using the ART (ask, respond, tell) communication technique to engage patients, and developing a shared care plan. The training may be conducted in formats such as, but not limited to: interactive simulations practicing the skills above, or didactic instructions on how to implement improvement action plans, monitor progress, and promote stability around improved clinician communication.",IA_CC_18,Care Coordination,medium,,,,boolean,2019,,medium,careCoordination,
+Use of certified EHR to capture patient reported outcomes,"use, etc.) or patient activation measures through use of certified EHR technology, containing this data in a separate queue for clinician recognition and review.",IA_BE_1,Beneficiary Engagement,medium,,,,boolean,2017,,medium,beneficiaryEngagement,
```

Comment:
this new description kind of looks like a typo bc it starts lowercase and has an ending parenthesis... shrug

Reply:
Thanks for catching this Kalvin! I checked with Sarah and she's uploaded an updated file. I'll make the necessary changes.